R/testrun_queries.R

```diff
@@ -0,0 +1,76 @@
+#' Testrun queries
+#'
+#' the database will just have two table, one that is defined
+#' by the provided tibble and the
+#' the queries are expected to be read from a file
+#' `queries.sql` in the directory `dir`
+#'
+#' @param df tibble to turn into an sql table
+#' @param dir directory to write the output to
+#' @param table_name. name of the table in sql database
+#'
+#' @export
+#'
+#' @examples
+#' #' \dontrun{
+#'   # command to call. this function
+#'   testrun_queries(ds$data, ds$dir, ds$name)
+#'   # example input file `queries.sql`
+#'   -- SELECT COUNT(*) from -- Wieviele Abstimmungsvorlagen gab es seit dem Jahr 1971?
+#'   SELECT COUNT(*) as anzahl_vorlagen_in_2000
+#'.  FROM abstimmungsvorlagen_seit_1971 as T
+#'.  JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
+#'.  WHERE S.name_de='Schweiz' AND S.country=TRUE
+#' }
+#' #' #' \dontrun{
+#'   testrun_queries(ds$data, ds$dir, ds$name)
+#' }
+testrun_queries <- function(df, dir, table_name) {
+  statbotdb <- DBI::dbConnect(RSQLite::SQLite(), "")
+  spatial_df <- readr::read_csv("data/const/spatial_unit_postgres.csv",
+                                show_col_types = FALSE)
```

Comment:
@philbosch that is the only suggestion that I don't understand: the pipeline directory already exists, since I moved the scripts into it: I have moved `scripts/A6.R` to `pipelines/A6/A6.R`: therefore `pipelines/A6` the directory for the queries and log files already exists. The same will be the case for the other pipeline directories.

Reply:
@sabinem you are right, I got that one confused. With a script the directory has to exist as well :D